client/devdocs/main.jsx

```diff
@@ -1,28 +1,28 @@
 /**
  * External dependencies
  */
-var React = require( 'react' ),
-	isFunction = require( 'lodash/isFunction' ),
-	classNames = require( 'classnames' );
+import React from 'react';
+import isFunction from 'lodash/isFunction';
 
 /**
  * Internal dependencies
  */
-var DocService = require( './service' ),
-	Card = require( 'components/card' ),
-	Main = require( 'components/main' ),
-	SearchCard = require( 'components/search-card' );
+import DocService from './service';
+import Card from 'components/card';
+import Main from 'components/main';
+import SearchCard from 'components/search-card';
+import { decodeEntities } from 'lib/formatting';
```

Comment:
Had you considered decoding entities server-side before sending the response? I don't recall if importing from the `client` directory is an issue server-side, but even if it is, you can import `he` directly for the decode ([reference](https://github.com/Automattic/wp-calypso/blob/master/client/lib/formatting/decode-entities/node.js)). An added advantage here is that the snippet cut-off would apply to the decoded text (…, a single character) versus the encoded text (`&hellip;`, 8 characters).